`npm i` fails on Windows

Regression after #2410:

```
> respec@24.27.1 prepare C:\Users\sasch\Documents\GitHub\respec
> npm run snyk-protect; npm run build:components

npm ERR! missing script: snyk-protect;
npm ERR!
npm ERR! Did you mean this?
npm ERR!     snyk-protect

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\sasch\AppData\Roaming\npm-cache\_logs\2019-07-25T02_25_35_640Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! respec@24.27.1 prepare: `npm run snyk-protect; npm run build:components`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the respec@24.27.1 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\sasch\AppData\Roaming\npm-cache\_logs\2019-07-25T02_25_35_727Z-debug.log
```

Probably an npm bug, but not sure this has ever been supported.

--- a/package.json
+++ b/package.json
@@ -82,7 +82,7 @@
     "copydeps": "node ./tools/copydeps.js",
     "karma": "karma start --single-run",
     "lint": "tsc -p src/jsconfig.json && eslint .",
-    "prepare": "npm run snyk-protect; npm run build:components",
+    "prepare": "npm run snyk-protect && npm run build:components",
     "release": "node ./tools/release.js",
     "server": "npm start",
     "snyk-protect": "snyk protect",
@@ -103,7 +103,7 @@
     "opencollective-postinstall": "^2.0.2",
     "prompt": "^1.0.0",
     "puppeteer": "1.19.0",
-    "snyk": "^2.0.0"
+    "snyk": "^1.199.0"
   },
   "prettier": {
     "trailingComma": "es5"

